src/CMakeLists.txt

```diff
@@ -5,8 +5,14 @@ project(minetest)
 INCLUDE(CheckIncludeFiles)
 INCLUDE(CheckLibraryExists)
 
+if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+	set(COMPILER_ABI_FLAG "-Wabi=11")
+else()
+	set(COMPILER_ABI_FLAG "-Wabi")
```

Comment:
`-Wabi` is a [no-op](https://clang.llvm.org/docs/DiagnosticsReference.html#wabi) for clang, so just remove this?